website/docs/r/cluster.html.markdown

```diff
@@ -291,6 +291,13 @@ Physical location of the region.
 * `sample_size_bi_connector` - (Optional) Number of documents per database to sample when gathering schema information. Defaults to 100. Available only for Atlas deployments in which BI Connector for Atlas is enabled.
 * `sample_refresh_interval_bi_connector` - (Optional) Interval in seconds at which the mongosqld process re-samples data to create its relational schema. The default value is 300. The specified value must be a positive integer. Available only for Atlas deployments in which BI Connector for Atlas is enabled.
 
+### Labels
+Containing key-value pairs that tag and categorize the database user. Each key and value has a maximum length of 255 characters.
```

Comment:
Contains key-value pairs that tag and categorize the cluster. Each key and value has a maximum length of 255 characters.